src/pages/checkout/checkout.js

```diff
@@ -317,6 +317,16 @@ const populateFields = () => {
   }
 };
 
+const stickyMoneybox = () => {
```

Comment:
It isn't immediately clear, from the function name, what this does. It doesn't seem to be a pure function (doesn't return anything, but it seems to manipulate something outside of itself). Is it only run once? Perhaps it shouldn't be a function then?

Reply:
The function gets called whenever this event happended `window.addEventListener("scroll", stickyMoneybox);`

Reply:
Any suggestion for the function name?

Reply:
Made the changes